### PR TITLE
wording for workloads. Added Autopatch notes

### PR DESCRIPTION
**Pull request by StephendillonMS**

### CODE REVIEW

File: memdocs/configmgr/comanage/workloads.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ ms.collection: highpri
 
 # Co-management workloads
 
-You don't have to switch the workloads, or you can do them individually when you're ready. Configuration Manager continues to manage all other workloads, including those workloads that you don't switch to Intune, and all other features of Configuration Manager that co-management doesn't support.
+You don't have to switch any of the workloads, but you can do them individually or several or all at once when you're ready. However, until you switch the workloads over to Intune, Microsoft Endpoint Configuration Manager continues to manage those workloads that you don't switch to Intune, alongside all other features of Configuration Manager that co-management doesn't support.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/comanage/workloads.md:18]:
```suggestion
You don't have to switch any of the workloads, but you can do them individually or several or all at the same time when you're ready. However, until you switch the workloads over to Intune, Microsoft Endpoint Configuration Manager continues to manage those workloads that you don't switch to Intune, along with all other features of Configuration Manager that co-management doesn't support.
```

File: memdocs/configmgr/comanage/workloads.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ ms.collection: highpri
...
 
-If you switch a workload to Intune, but later change your mind, you can switch it back to Configuration Manager.
+If you switch a workload to Intune, but later change your mind, you can switch it back to Configuration Manager, although there maye be an impact for example Windows and Office versions will remain at a later version if installed by Intune.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/comanage/workloads.md:20]:
```suggestion
If you switch a workload to Intune, but later change your mind, you can switch it back to Configuration Manager, although there might be an impact, for example, Windows and Office versions will remain at a later version if installed by Intune.
```

File: memdocs/configmgr/comanage/workloads.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ For more information on the Intune feature, see [Use compliance policies to set
 
 ## Windows Update policies
 
-Windows Update for Business policies let you configure deferral policies for Windows 10 or later feature updates or quality updates for Windows 10 or later devices managed directly by Windows Update for Business.
+Windows Update for Business policies let you configure deferral policies for Windows 10 or later feature updates or quality updates for Windows 10 or later devices managed directly by Windows Update for Business. Note: This workload needs to be managed by Intune for the Autopatch feature to work for these devices.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/comanage/workloads.md:46]:
```suggestion
Windows Update for Business policies let you configure deferral policies for Windows 10 or later feature updates or quality updates for Windows 10 or later devices managed directly by Windows Update for Business. Note that this workload needs to be managed by Intune for the Windows Autopatch feature to work on these devices.
```

File: memdocs/configmgr/comanage/workloads.md [hunk 4]
```diff
@@ -94,7 +94,7 @@ For more information on the Intune feature, see [Windows 10 (and later) settings
...
+You can still deploy settings from Configuration Manager to co-managed devices even though Intune is the device configuration authority. This exception might be used to configure settings that your organization requires but aren't yet available in Intune. Specify this exception on a [Configuration Manager configuration baseline](../compliance/deploy-use/create-configuration-baselines.md). Enable the option to **Always apply this baseline even for co-managed clients** when creating the baseline. You can change it later on the **General** tab of the properties of an existing baseline. Note: This workload needs to be managed by Intune for the Autopatch feature to work for these devices.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/comanage/workloads.md:97]:
```suggestion
You can still deploy settings from Configuration Manager to co-managed devices even though Intune is the device configuration authority. This exception might be used to configure settings that your organization requires but aren't yet available in Intune. Specify this exception on a [Configuration Manager configuration baseline](../compliance/deploy-use/create-configuration-baselines.md). Enable the option to **Always apply this baseline even for co-managed clients** when creating the baseline. You can change it later on the **General** tab of the properties of an existing baseline. Note that this workload needs to be managed by Intune for the Windows Autopatch feature to work on these devices.
```

File: memdocs/configmgr/comanage/workloads.md [hunk 5]
```diff
@@ -124,6 +124,7 @@ Updates can be managed using either of the following features:
 
 - [Use Update Channel and Target Version settings to update Microsoft 365 with Microsoft Intune Administrative Templates](../../intune/configuration/administrative-templates-update-office.md)
 - [Manage Microsoft 365 Apps with Configuration Manager](../sum/deploy-use/manage-office-365-proplus-updates.md).
+Note: This workload needs to be managed by Intune for the Autopatch feature to work for these devices.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/comanage/workloads.md:127]:
```suggestion
Note that this workload needs to be managed by Intune for the Windows Autopatch feature to work on these devices.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @StephendillonMS, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by aczechowski — APPROVED** [review]
Thanks @StephendillonMS for the edit and thank you for your first contribution to memdocs! 🎉 

I confirmed the info with the Autopatch team. I made a few adjustments to the wording and used the note format in a couple of the instances (not all...because one section already has a lot of notes). And included a link to the Autopatch prerequisites that also discusses this topic.

**Comment by aczechowski** [comment]
#sign-off

**Pull request merged**